smart_contracts/contracts_as/test/groups/assembly/index.ts

```diff
@@ -113,9 +113,9 @@ export function call_restricted_entry_points(): void {
 
 function createGroup(packageHash: Uint8Array): URef {
   let key = Key.create(CLValue.fromU64(0));
```

Comment:
Same here about the variable rename

Reply:
Good point - I did a full pass on other contracts as well in c6276b99e5574cb0ec38075f99b9472707b84877